src/index.js

```diff
@@ -94,9 +94,15 @@ class Spreadsheet {
     return this.dataSet.map((it) => it.getData());
   }
 
-  cellText(ri, ci, text, force = false, sheetIndex = 0) {
+  cellText(ri, ci, text, force = false, saveHistory = false, sheetIndex = 0) {
```

Comment:
why is this per default `false`, should it not be `true`?

Reply:
In our VDI app, we frequently utilize this function to write error and warning texts in cells that are not required to be saved in the history. Approximately 95% of the time, we do not need these texts to be saved, while only about 5% of the time, we actually require them to be preserved. Therefore, it is more efficient and storage-friendly to have this setting as false by default.

Reply:
Okay, I agree. It's changed now.